pipelines/batch/src/main/java/com/google/fhir/analytics/JdbcResourceWriter.java

```diff
@@ -83,6 +85,18 @@ private static void createSingleTable(DataSource dataSource, String createStatem
     }
   }
 
+  private static void createIdIndex(DataSource dataSource, String tableName) throws SQLException {
```

Comment:
I am assuming the reason for defining just an index and not a `UNIQUE INDEX` nor a `PRIMARY KEY` is that the target records in the table may have multiple rows with same `id` due to flattening.